repos/system_upgrade/common/actors/addupgradebootentry/tests/unit_test_addupgradebootentry.py

```diff
@@ -273,7 +273,7 @@ def test_get_rdlvm_arg_values(monkeypatch):
 
     args = addupgradebootentry._get_rdlvm_arg_values()
 
-    assert args == ['A', 'B']
+    assert tuple(args) == ('A', 'B')
```

Comment:
`_get_rdlvm_arg_values` should be already returning a `tuple`, so shouldn't you be testing:
```suggestion
    assert args == ('A', 'B')
```

otherwise you're asserting that the return value of `_get_rdlvm_arg_values` can be converted to a tuple matching `(A, B)`

Reply:
good catch!